lib/fixture_factory/errors.rb

```diff
@@ -30,11 +30,12 @@ def initialize(method_name)
   end
 
   class WrongClassError < Error
-    def initialize(class_name)
+    def initialize(klass)
+      location, line_number = klass.source_location
```

Comment:
This is a proc right? It might make more sense if we named this `class_proc` or `lazy_class`.